src/mem/mem_ctrl.cc

```diff
@@ -517,16 +1202,23 @@ MemCtrl::processRespondEvent()
         }
     } else {
         // it is not a split packet
```

Comment:
Add a panic to the above code in case you have a split packet since this code doesn't handle it correctly for now